secure_message_v2/application.py

```diff
@@ -8,23 +7,27 @@
 
 from secure_message_v2.models import models
 from secure_message_v2.views.info import info_bp
+from secure_message_v2.views.messages import messages_bp
+from secure_message_v2.views.threads import threads_bp
 
 logger = wrap_logger(logging.getLogger(__name__))
 
 
-def create_app(config=None):
+def create_app():
     app = Flask(__name__)
     app.name = "ras-rm-secure-message-v2"
     logger.info("Creating app", name=app.name)
-    app_config = f"config.{config or os.getenv('APP_SETTINGS', 'Config')}"
+    app_config = "config.Config"
     app.config.from_object(app_config)
 
     app.register_blueprint(info_bp, url_prefix="/info")
+    app.register_blueprint(messages_bp, url_prefix="/messages")
+    app.register_blueprint(threads_bp, url_prefix="/threads")
```

Comment:
We should consider an API.md or extending the README.md to document the usage of these